README.md

```diff
@@ -40,42 +40,44 @@ And then you can use it this way:
 ## Usage
 
 The boring generator introduces following generators:
-- [Install Tailwind CSS](https://www.boringgenerators.com/blog/2020-10-18-install-tailwind/): `rails generate boring:tailwind:install`
-- [Install Bootstrap](https://www.boringgenerators.com/blog/2020-11-15-install-bootstrap/): `rails generate boring:bootstrap:install`
-- Install JQuery: `rails generate boring:jquery:install`
-- [Install FontAwesome via Yarn](https://www.boringgenerators.com/blog/2021-02-28-install-fontawesome-yarn/): `rails generate boring:font_awesome:yarn:install`
-- [Install FontAwesome via RubyGems](https://www.boringgenerators.com/blog/2021-02-23-install-fontawesome/): `rails generate boring:font_awesome:ruby_gem:install`
-- Install Bullet: `rails generate boring:bullet:install`
-- Install Audit gems(bundler-audit, ruby_audit): `rails generate boring:audit:install`
-- Install Pry gems for easy debugging: `rails generate boring:pry:install`
-- Install Active Storage for Google Cloud Service: `rails generate boring:active_storage:google:install`
+
+- Build Favicon: `rails generate boring:favicon:build --application_name=<application_name> --favico_letter=<favico_letter> --primary_color=<color>`
 - Install Active Storage for AWS: `rails generate boring:active_storage:aws:install`
 - Install Active Storage for Azure: `rails generate boring:active_storage:azure:install`
+- Install Active Storage for Google Cloud Service: `rails generate boring:active_storage:google:install`
```

Comment:
This looks nice but I think it would be hard for us to maintain the order of generators as we add more of them. Let's not worry about it for future additions.